package-lock.json

```diff
@@ -2274,7 +2274,7 @@
       "dependencies": {
         "es6-promise": {
           "version": "3.0.2",
-          "resolved": "https://registry.npmjs.org/es6-promise/-/es6-promise-3.0.2.tgz",
+          "resolved": "http://registry.npmjs.org/es6-promise/-/es6-promise-3.0.2.tgz",
```

Comment:
Why is there the change from https to http? (same with readable-stream dep)

Reply:
@hmlnarik because that's the standard in new versions of npm. Do you see any blockers?

Reply:
Strange. So while the rest of the world is going to https always NPM is for some reason going backwards?

Reply:
No blockers, the switch from https to unsecured http just seemed a typo to me (particularly with almost of the other dependencies still using https)

Reply:
@hmlnarik @stianst let me rephrase myself. Not a standard, but known bug https://github.com/npm/npm/issues/20719